src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
I don't really like it because it implies the existence of an access inspector that is not magic aware, and that's only going to exist in the magespec repo.

Sometimes if you really can't name something, then the implementation might not be right. Maybe we need several Inspectors for the specific tasks that the current one is doing, like `MagicAccessInspector`, `ReflectionAccessInspector`, `GenericAccessInspector`. They can be iterated over until one returns true, and then all Magespec has to do is remove the one for magic methods from the list.